### PR TITLE
fix: graphql introspection with renamed query types

### PR DESCRIPTION
**Pull request by devsergiy**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: packages/sdk/src/graphql/schema.ts [hunk 1]
```diff
@@ -21,6 +22,32 @@ export const cleanupSchema = (schema: GraphQLSchema, introspection: GraphQLIntro
 	const queryTypeName = schema.getQueryType()?.name;
 	const mutationTypeName = schema.getMutationType()?.name;
 	const subscriptionTypeName = schema.getSubscriptionType()?.name;
+
+	const replaceOperationTypeName = (name: NameNode): NameNode => {
```
Comment (Aenimus) [inline packages/sdk/src/graphql/schema.ts:26]: Would a switch be cleaner?

Comment (devsergiy) [reply packages/sdk/src/graphql/schema.ts:26]: yep, updated

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,12 @@
 all: check-setup
 # Bootstrap pnpm workspace
-	./scripts/pnpm.sh
+	make pnpm
 # prepare and install engine
 	make engine-dev
 
+pnpm:
```
Comment (StarpTech) [inline Makefile:7]: Could you give it a more descriptive name like `setup-pnpm`?

**Review by StarpTech — CHANGES_REQUESTED** [review]

**Review by Aenimus — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (devsergiy) [reply Makefile:7]: is it a setup?

maybe `pnpm-build` or `bootstrap-pnpm`?

Comment (StarpTech) [reply Makefile:7]: Yeah, I like :+1:

Comment (devsergiy) [reply Makefile]: done

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**